sript.js

```diff
@@ -196,4 +193,47 @@ form.addEventListener('submit', (event) => {
     email.classList.add('error-shadow');
     error.style.display = 'block';
   }
-});
\ No newline at end of file
+});
+
+// Keep Data
+
+const fullName = document.getElementById('name');
+const comment = document.getElementById('comment');
+
+function keep() {
+  const input = {
+    inputName: fullName.value,
+    inputEmail: email.value,
+    inputComment: comment.value,
+  };
+  localStorage.setItem('input', JSON.stringify(input));
+}
+
+let formObject = JSON.parse(localStorage.getItem('input'));
+
+if (formObject === undefined) {
+  formObject = {
+    fullName: '',
+    email: '',
+    comment: '',
+  };
+  keep();
+}
+
+fullName.value = formObject.fullName;
+fullName.addEventListener('input', (event) => {
+  formObject.fullName = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
+
+email.value = formObject.email;
+email.addEventListener('input', (event) => {
+  formObject.email = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
+
+comment.value = formObject.comment;
+comment.addEventListener('input', (event) => {
+  formObject.comment = event.target.value;
+  localStorage.setItem('input', JSON.stringify(formObject));
+});
```

Comment:
- [ ] Nice work having a single object to save in the local storage :+1: However the data is not saving in the local storage. Kindle make sure when the user changes the content of any input field, the data is saved to the local storage to follow the project requirement. 
**HIT: It will be nice to listen to a `change` event to save the `formObject` to the localstorage**

E.g.:
```js
comment.addEventListner('input', () => {});
```

Reply:
Thank you for your review.

I am not quite understanding. Should I just change the event listeners I gave the values to `change` instead of `input`?